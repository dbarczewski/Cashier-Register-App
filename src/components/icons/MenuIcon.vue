<template>
  <svg width="28" height="28" viewbox="0 0 28 28">
    <g fill="#FDFEFF" fill-rule="nonzero">
      <circle pid="0" cx="4" cy="4" r="4" />
      <circle pid="1" opacity=".9" cx="14" cy="4" r="4" />
      <circle pid="2" opacity=".8" cx="24" cy="4" r="4" />
      <circle pid="3" opacity=".7" cx="4" cy="14" r="4" />
      <circle pid="4" opacity=".6" cx="14" cy="14" r="4" />
      <circle pid="5" opacity=".5" cx="24" cy="14" r="4" />
      <circle pid="6" opacity=".4" cx="4" cy="24" r="4" />
      <circle pid="7" opacity=".3" cx="14" cy="24" r="4" />
      <circle pid="8" opacity=".2" cx="24" cy="24" r="4" />
    </g>
  </svg>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({
  name: "MenuIcon",
});
</script>
