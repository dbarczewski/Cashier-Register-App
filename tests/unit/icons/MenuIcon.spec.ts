import { shallowMount } from "@vue/test-utils";
import MenuIcon from "@/components/icons/MenuIcon.vue";

describe("icons/MenuIcon", () => {
  it("renders component correctly", () => {
    const wrapper = shallowMount(MenuIcon);
    expect(wrapper.element).toMatchSnapshot();
  });
});
