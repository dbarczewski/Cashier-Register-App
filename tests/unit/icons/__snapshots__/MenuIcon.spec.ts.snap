// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons/MenuIcon renders component correctly 1`] = `
<svg
  height="28"
  viewbox="0 0 28 28"
  width="28"
>
  <g
    fill="#FDFEFF"
    fill-rule="nonzero"
  >
    <circle
      cx="4"
      cy="4"
      pid="0"
      r="4"
    />
    <circle
      cx="14"
      cy="4"
      opacity=".9"
      pid="1"
      r="4"
    />
    <circle
      cx="24"
      cy="4"
      opacity=".8"
      pid="2"
      r="4"
    />
    <circle
      cx="4"
      cy="14"
      opacity=".7"
      pid="3"
      r="4"
    />
    <circle
      cx="14"
      cy="14"
      opacity=".6"
      pid="4"
      r="4"
    />
    <circle
      cx="24"
      cy="14"
      opacity=".5"
      pid="5"
      r="4"
    />
    <circle
      cx="4"
      cy="24"
      opacity=".4"
      pid="6"
      r="4"
    />
    <circle
      cx="14"
      cy="24"
      opacity=".3"
      pid="7"
      r="4"
    />
    <circle
      cx="24"
      cy="24"
      opacity=".2"
      pid="8"
      r="4"
    />
  </g>
</svg>
`;
